sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -397,6 +419,8 @@ async Task<bool> OnBreakpointHit (SessionId sessionId, JObject args, Cancellatio
 			var res = await SendMonoCommand (sessionId, MonoCommands.GetCallStack(), token);
 			var orig_callframes = args? ["callFrames"]?.Values<JObject> ();
 			var context = GetContext (sessionId);
+			JObject data = null;
```

Comment:
We should probably rename `OnBreakpointHit` to `OnPause` now, as this handles exceptions too?

Reply:
Done!